docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -22,85 +33,230 @@
+schedule.
 
-\section{Confidential Information?}
+% \section{Confidential Information?}
 
-\wss{State whether your project has confidential information from industry, or
-not.  If there is confidential information, point to the agreement you have in
-place.}
+% \wss{State whether your project has confidential information from industry, or
+% not.  If there is confidential information, point to the agreement you have in
+% place.}
 
-\wss{For most teams this section will just state that there is no confidential
-information to protect.}
-\section{IP to Protect}
+% \wss{For most teams this section will just state that there is no confidential
+% information to protect.}
+% \section{IP to Protect}
 
-\wss{State whether there is IP to protect.  If there is, point to the agreement.
-All students who are working on a project that requires an IP agreement are also
-required to sign the ``Intellectual Property Guide Acknowledgement.''}
+% \wss{State whether there is IP to protect.  If there is, point to the agreement.
+% All students who are working on a project that requires an IP agreement are also
+% required to sign the ``Intellectual Property Guide Acknowledgement.''}
 
 \section{Copyright License}
-
-\wss{What copyright license is your team adopting.  Point to the license in your
-repo.}
+The project will be licensed under the MIT License, allowing for open-source use, modification, 
+and distribution with minimal restrictions. Found \href{https://github.com/lilweege/SyntaxSentinels/blob/main/LICENSE}{here}.
 
 \section{Team Meeting Plan}
 
-\wss{How often will you meet? where?}
+The team will meet for at least 4 hours every week, 2 hours during the allotted tutorial 
+time on Monday at 2:30 pm. Precise location will be open but the meeting will be virtually, 
+otherwise on McMaster University campus. the other 2 hours allotted would be done virtually over the weekend.
 
-\wss{If the meeting is a physical location (not virtual), out of an abundance of
-caution for safety reasons you shouldn't put the location online}
 
-\wss{How often will you meet with your industry advisor?  when?  where?}
+Meetings will be scheduled with our industry advisor depending on their schedule virtually. 
+The plan is to meet for at least 4 hours every month for guidance and to confirm that the team 
+is on the right track. We hope to meet with our advisor for at least 1 hour at the end of every 
+month for a recap. In person meetings with our advisor will be conducted depending on their availability.
 
-\wss{Will meetings be virtual?  At least some meetings should likely be
-in-person.}
+\subsection*{Roles}
 
-\wss{How will the meetings be structured?  There should be a chair for all meetings.  There should be an agenda for all meetings.}
+\begin{itemize}
+    \item \textbf{Chair}: Dennis will chair all meetings.
+    \item \textbf{Minute-taker}: Mohsin will take minutes during all meetings.
+\end{itemize}
```

Comment:
These roles are very surface level and non-technical. We should have some technical designation. I'll work on this now and push a commit

Reply:
these are jusdt for meeting roles